src/lib/Components/Bidding/Screens/BillingAddress.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { ScrollView } from "react-native"
+
+import { Serif16 } from "../Elements/Typography"
+
+import { BiddingThemeProvider } from "../Components/BiddingThemeProvider"
+import { Button } from "../Components/Button"
+import { Container } from "../Components/Containers"
+import { Input } from "../Components/Input"
+import { Title } from "../Components/Title"
+
+export class BillingAddress extends React.Component {
+  render() {
+    return (
+      <BiddingThemeProvider>
+        <ScrollView>
+          <Container>
+            <Title mt={0} mb={6}>
+              Your billing address
+            </Title>
+
+            <Serif16 mb={2}>Full name</Serif16>
+            <Input placeholder="Enter your full name" mb={4} />
+
+            <Serif16 mb={2}>Address line 1</Serif16>
+            <Input placeholder="Enter your street address" mb={4} />
+
+            <Serif16 mb={2}>Address line 2 (optional)</Serif16>
+            <Input placeholder="Enter your apt, floor, suite, etc." mb={4} />
+
+            <Serif16 mb={2}>City</Serif16>
+            <Input placeholder="Enter city" mb={4} />
+
+            <Serif16 mb={2}>State, Provice, or Region</Serif16>
```

Comment:
Eep didn't notice this before! But `Provice` --> `Province`

Reply:
😩

Reply:
I use an amazing spellchecker! Maybe there's a port for webstorm? https://github.com/Jason-Rev/vscode-spell-checker

Reply:
IntelliJ does check spelling! I should just pay more attention to its suggestions...